<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_Dealer">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Dealer</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>175</y>
      <width>121</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Player 1: $xxx</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_Bet_1">
       <property name="text">
        <string>Bet:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox_1"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_Show_1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Player_1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_1A">
       <item>
        <widget class="QPushButton" name="pushButton_Double_1">
         <property name="text">
          <string>Double</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Split_1">
         <property name="text">
          <string>Split</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_1B">
       <item>
        <widget class="QPushButton" name="pushButton_Hit_1">
         <property name="text">
          <string>Hit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Stand_1">
         <property name="text">
          <string>Stand</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_Play">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>185</y>
      <width>121</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Player 2: $xxx</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_Bet_2">
       <property name="text">
        <string>Bet:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox_2"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_Show_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Player_1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2A">
       <item>
        <widget class="QPushButton" name="pushButton_Double_2">
         <property name="text">
          <string>Double</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Split_2">
         <property name="text">
          <string>Split</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2B">
       <item>
        <widget class="QPushButton" name="pushButton_Hit_2">
         <property name="text">
          <string>Hit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Stand_2">
         <property name="text">
          <string>Stand</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>121</width>
      <height>256</height>
     </rect>
    </property>
    <property name="title">
     <string>Player 3: $xxx</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_Bet_3">
       <property name="text">
        <string>Bet:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox_3"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_Show_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Player_1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3A">
       <item>
        <widget class="QPushButton" name="pushButton_Double_3">
         <property name="text">
          <string>Double</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Split_3">
         <property name="text">
          <string>Split</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3B">
       <item>
        <widget class="QPushButton" name="pushButton_Hit_3">
         <property name="text">
          <string>Hit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Stand_3">
         <property name="text">
          <string>Stand</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>121</width>
      <height>256</height>
     </rect>
    </property>
    <property name="title">
     <string>Player 4: $xxx</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_Bet_4">
       <property name="text">
        <string>Bet:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox_4"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_Show_4">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Player_1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4A">
       <item>
        <widget class="QPushButton" name="pushButton_Double_4">
         <property name="text">
          <string>Double</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Split_4">
         <property name="text">
          <string>Split</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4B">
       <item>
        <widget class="QPushButton" name="pushButton_Hit_4">
         <property name="text">
          <string>Hit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Stand_4">
         <property name="text">
          <string>Stand</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>167</y>
      <width>111</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Player 5: $xxx</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="label_Bet_5">
       <property name="text">
        <string>Bet:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox_5"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_Show_5">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Player_1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_5A">
       <item>
        <widget class="QPushButton" name="pushButton_Double_5">
         <property name="text">
          <string>Double</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Split_5">
         <property name="text">
          <string>Split</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_5B">
       <item>
        <widget class="QPushButton" name="pushButton_Hit_5">
         <property name="text">
          <string>Hit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Stand_5">
         <property name="text">
          <string>Stand</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Dealer:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>674</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStart">
    <property name="title">
     <string>Start</string>
    </property>
    <addaction name="actionSomething_Here"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionhit_on_soft_17"/>
    <addaction name="actionInsurance"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuStart"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionhit_on_soft_17">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hit on soft 17</string>
   </property>
  </action>
  <action name="actionInsurance">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Insurance</string>
   </property>
  </action>
  <action name="actionSomething_Here">
   <property name="text">
    <string>Something Here</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
